endpoints.json

```diff
@@ -42,6 +42,21 @@
           "comment_count": 6
         }
     }
+  },
+  "GET /api/articles/:article_id/comments": {
+    "description": "serves an array of comments related to an article",
+    "queries": [],
+    "exampleResponse": {
+      "comments": [
+        {
+          "comment_id": 1,
+          "body": "Seafood substitutions are increasing",
+          "votes": "cooking",
+          "author": "weegembump",
+          "created_at": "2018-05-30T15:59:13.341Z",
+          "article_id": 0
+        }
```

Comment:
This is really good and how it should look like, but you are not actually sending back an object with a key of comments with the array of comments, you are directly sending back the array